stable/yugaware/templates/service.yaml

```diff
@@ -227,10 +227,3 @@ spec:
             mountPath: /opt/certs/
             readOnly: true
           {{- end }}
-        - name: dnsmasq
-          image: {{ include "full_image" (dict "containerName" "dnsmasq" "root" .) }}
```

Comment:
there is an entry corresponding to this in values.yaml which also needs to be removed.